.github/workflows/ci.yml

```diff
@@ -76,12 +76,41 @@ jobs:
         run: |
           PACKAGE_VERSION=$(cargo metadata --format-version=1 --no-deps | jq -e -r '.packages[] | select(.name == "'"$CRATE"'") | .version')
           SPECIFIED_VERSION=$(tomlq "workspace.dependencies.$CRATE.version" --file ./Cargo.toml)
-          
+
           echo "Package version: $PACKAGE_VERSION";
           echo "Specified version: $SPECIFIED_VERSION";
 
           test "$PACKAGE_VERSION" = "$SPECIFIED_VERSION"
 
+  wasm_tests:
+    name: Run all WASM tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: dtolnay/rust-toolchain@stable
+        with:
+          target: wasm32-unknown-unknown
+
+      - uses: taiki-e/cache-cargo-install-action@v1
+        with:
+          tool: wasm-pack@0.12.0
+
+      - name: Install Google Chrome
+        env:
+          CHROME_VERSION: "114.0.5735.106"
+        run: |
+          curl -o /tmp/google-chrome-stable_amd64.deb https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_VERSION}-1_amd64.deb
+          sudo dpkg -i /tmp/google-chrome-stable_amd64.deb
+
+      - name: Install chromedriver
+        uses: nanasess/setup-chromedriver@v2
+        with:
+          chromedriver-version: '114.0.5735.90'
```

Comment:
Can we unify these two versions into one variable?

As far as I understand, these need to match, correct?

Reply:
Only the major version needs to much. Unfortunately chromedriver is not released on every Chrome release, however we can use an older version of Chrome if you prefer.

Reply:
I am okay with using a slightly older version for the benefit of only having to update a single one. We'll need to look for a particular chrome-driver release anyway so we can just update to that then.

Reply:
done